public/app/plugins/datasource/postgres/datasource.ts

```diff
@@ -185,11 +185,23 @@ export class PostgresDatasource extends DataSourceWithBackend<PostgresQuery, Pos
   }
 
   getVersion(): Promise<any> {
-    return lastValueFrom(this._metaRequest("SELECT current_setting('server_version_num')::int/100"));
+    return lastValueFrom(this._metaRequest("SELECT current_setting('server_version_num')::int/100")).then((value) => {
```

Comment:
So I was looking at these changes and wondered why are these needed. So we had a code that checked for version in the config page code that was not working. What do you all think, do we need that? @oscarkilhed @scottlepp

Reply:
looks like that is/was used for feature detection/validation in postgres.  i'm a bit reluctant to remove it since it was already there, but not sure if it is absolutely necessary

Reply:
I added back that code that found the version and set timescale db to true if it was timescale.